cluster/restore-from-backup.sh

```diff
@@ -76,9 +76,9 @@ wait_for_etcd_up() {
   # {"health": "true"} on /health endpoint in healthy case.
   # However, we should come with a regex for it to avoid future break.
   health_ok="{\"health\": \"true\"}"
-  for i in $(seq 120); do
+  for _ in $(seq 120); do
```

Comment:
these `i` loop variables are unused placeholders to accomplish the seq loop, `_` is clearer that we intend to ignore them and makes shellcheck happy.